__tests__/Signature.js

```diff
@@ -5,34 +5,34 @@ var ReactDOM = require('react-dom');
 var TestUtils = require('react-addons-test-utils');
 
 var Signature = require('../src/js/Signature.jsx');
-var fs = require('fs');
 var api = require('../src/js/api');
 
+var code = 13;
 api.postSignature = jest.fn(function(cb){
-  cb({
+  cb(null, {
     status: {
-      code: state,
-      message: ""
+      code: code,
+      message: ''
     },
     timestamp: Date.now(),
-    receipt: 'somehash'
+    receipt: 'dc9e5827abb678f54103e7b89435abf9b36648797ebb6516a52ab33ab4e46cee'
   });
+  code = code === 13 ? 12 : 13;
 });
 
 describe('irs report', function(){
-  var changeHandlerTrue = function(e){
-    expect(e.target.checked).toBeTruthy();
+  var changeHandlerTrue = function(err, status){
```

Comment:
I like the way you did things in the IRS test, for [example](https://github.com/cfpb/hmda-platform-ui/pull/142/files#diff-b28d2400315fa2e86b0451fc245b96bbR56). Both the function names and declaring them inside the `it()`.
Maybe do that here too?